package.json

```diff
@@ -5,6 +5,7 @@
     "istanbul": "^0.4.2",
     "lerna": "2.0.0-beta.15",
     "mocha": "^2.5.3",
+    "mocha-phantomjs-core": "^1.3.1",
```

Comment:
replace mocha-phantomjs so that we don't download prebuilt binary with segfaults